Cargo.toml

```diff
@@ -6,7 +6,9 @@ members = [
     "crates/db-arch",
     "crates/db-boot",
     "crates/dbs-device",
+    "crates/dbs-interrupt",
 ]
 
 [patch.crates-io]
 db-boot = { path = "crates/db-boot" }
+dbs-device = { path = "crates/dbs-device" }
```

Comment:
The whole [patch.crates-io] will be removed in later PR. You could look at PR https://github.com/openanolis/dragonball-sandbox/pull/60 for more information about crates in workspace.